BLSQ/iaso
# POLIO-1583 POLIO-1584 POLIO-1585: Calendar -Non Polio campaigns in orange

Non polio campaigns should have another specific color.
Also campaign legend is a bit buggy:

https://github.com/BLSQ/iaso/assets/12494624/0e995db4-8c5b-4e25-a4a8-635e3e15a82a


Related JIRA tickets : POLIO-1583 POLIO-1584 POLIO-1585

## Self proofreading checklist


## Changes

- remove campaign color for merged shapes as not used anymore. If no vaccine display theme secondary color (orange for Poio)
- apply same logic  on regular shapes
- add Other in map legend
- fix campaigns legend styling issues
- display same color while selecting a scope on non polio campaigns
- adapt celle background color to the same color if a scope exists and no vaccine is used

## How to test

You need a polio campaign with a scope, a non polio one also with scope and a third one without scope.
Make it display at the same time on the calendar for tests purposes.
The polio campaign should use vaccine scope colors on the cells and on the map
The non polio one should use secondary color
The third one should not be displayed on the map but should be in grey.

Make sure you have the same behaviour while zooming on shapes and use non merged shapes, displaying borders according to campaigns legend

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/c5ffa54b-a350-492b-8f73-d10444e918e2

<img width="1654" alt="Screenshot 2024-06-21 at 13 23 29" src="https://github.com/BLSQ/iaso/assets/12494624/e5d96d24-3265-4a82-936c-c9e85c321dc5">
<img width="1650" alt="Screenshot 2024-06-21 at 13 23 22" src="https://github.com/BLSQ/iaso/assets/12494624/b11fdf68-9d04-4815-ad77-7080c26cd2bd">
